DataAccess/DataAccess.Postgre/PostgreAdapter.hpp

```diff
@@ -20,7 +20,7 @@ class PostgreAdapter final : public IAdapter
 private:
     inline static std::mutex                        _staticMutex{};
     inline static std::shared_ptr<PostgreAdapter>   _instance{};
-    inline static constexpr std::string_view        _defaultOptions = DBOptions::real;
+    inline static constexpr std::string_view        _defaultOptions = DBOptions::test;
```

Comment:
You shouldn't make any changes at files not related with tests.

Reply:
I did this for views of tests